test/studies/cholesky/jglewis/version2/elemental/scalar_inner_product_cholesky.chpl

```diff
@@ -92,4 +92,3 @@ module scalar_inner_product_cholesky  {
 
   }
 }
-	
```

Comment:
I think you eliminated "no newline at end of file". If this is something else, please double-check.